processing/src/test/java/io/druid/segment/filter/ExtractionDimFilterTest.java

```diff
@@ -47,18 +51,22 @@
   );
 
   private static final Map<String, String> EXTRACTION_VALUES = ImmutableMap.of(
-      "foo1","extractDimVal"
+      "foo1", "extractDimVal"
   );
 
   private static ImmutableBitmap foo1BitMap;
+
   @BeforeClass
-  public static void setupStatic(){
+  public static void setupStatic()
+  {
     final ConciseSet conciseSet = new ConciseSet();
     conciseSet.add(1);
-    foo1BitMap = new WrappedConciseBitmap(conciseSet);
+    foo1BitMap = new WrappedImmutableConciseBitmap(ImmutableConciseSet.newImmutableFromMutable(conciseSet));
   }
+
   private static final BitmapIndexSelector BITMAP_INDEX_SELECTOR = new BitmapIndexSelector()
   {
+    private final ConciseBitmapFactory factory = new ConciseBitmapFactory();
```

Comment:
at some point we should make tests relying on bitmapfactory parameterized, so we test all types of bitmaps.